### PR TITLE
ACMS-1593: Updating contrib modules to latest release as per D10 comp…

### PR DESCRIPTION
**Pull request by apathak18**
…atible.

**Motivation**

Fixes ACMS-1593

**Proposed changes**

**Alternatives considered**

**Testing steps**

**Merge requirements**
- [ ] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

File: modules/acquia_cms_headless/composer.json [hunk 1]
```diff
@@ -74,6 +74,10 @@
         }
     },
     "repositories": {
+        "drupal": {
```
[the remark this answers is not in the document]
Comment (apathak18) [reply modules/acquia_cms_headless/composer.json]: Done

File: modules/acquia_cms_starter/composer.json [hunk 2]
```diff
@@ -22,5 +22,11 @@
             "phpro/grumphp-shim": true,
             "webdriver-binary/binary-chromedriver": true
         }
+    },
```
[the remark this answers is not in the document]
Comment (apathak18) [reply modules/acquia_cms_starter/composer.json]: Done

File: modules/acquia_cms_search/composer.json [hunk 3]
```diff
@@ -8,8 +8,8 @@
         "drupal/acquia_cms_common": "2.0.x-dev",
         "drupal/acquia_search": "^3.1",
         "drupal/collapsiblock": "^4.0",
-        "drupal/facets": "2.0.5",
-        "drupal/facets_pretty_paths": "^1.3",
+        "drupal/facets": "^2.0",
```
Comment (rajeshreeputra) [inline modules/acquia_cms_search/composer.json:11]: update to **"drupal/facets": "^2.0.6"**, OR **"drupal/facets": "^3.0"**, accordingly

Comment (apathak18) [reply modules/acquia_cms_search/composer.json]: Done

**Review by rajeshreeputra — APPROVED** [review]
LGTM

**Comment by rajeshreeputra** [comment]
@vishalkhode1 can we merge this in develop branch?

**Pull request merged**